From 7b256829ce730ab93b76e4f4f2e72f0132122ba1 Mon Sep 17 00:00:00 2001
From: Mamoru TASAKA <mtasaka@fedoraproject.org>
Date: Mon, 3 May 2021 14:43:56 +0900
Subject: [PATCH] xscreensaver-text use en_US locale for lscpu

Downstream bug: https://bugzilla.redhat.com/show_bug.cgi?id=1956089
xscreensaver-text in version 6.00 adds CPU model detection, using
lscpu command utility and using regex.
Since regex string is locale-independent, the result of lscpu should be
locale-independent. For now, let's use en_US "LANGUAGE".
---
 hacks/xscreensaver-text | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hacks/xscreensaver-text b/hacks/xscreensaver-text
index 421fae7..645bdd0 100755
--- a/hacks/xscreensaver-text
+++ b/hacks/xscreensaver-text
@@ -428,7 +428,7 @@ sub output() {
     # we're running on.  This info also exists in various /proc/ files,
     # but if /proc/ exists, lscpu probably exists as well.
     #
-    my $cpu = `lscpu 2>&-`;
+    my $cpu = `env LANGUAGE=en_US lscpu 2>&-`;
     if ($cpu) {
       my ($model) = ($cpu =~ m/^Model name:\s*(.*)$/mi);
      #my ($arch)  = ($cpu =~ m/^Architecture:\s*(.*)$/mi);
-- 
2.31.1

